test/Main.hs

```diff
@@ -75,3 +78,4 @@ main = defaultMain $ testGroup "lsm-tree"
     , Test.System.Posix.Fcntl.NoCache.tests
     , Test.Data.Arena.tests
     ]
+  Control.RefCount.checkForgottenRefs
```

Comment:
Unfortunately not. It will tell us where the ref was originally allocated.

This is why we also specifically include `checkForgottenRefs` into the state machine runner.

Doing it for every property test would be rather invasive. It could be done on demand if test fail.

Reply:
Ah, slightly less nice, but still good to have as a last resort.

Reply:
And we could use `hlint` to enforce using the custom version

Reply:
We could do it, but I think it's not worth it. It could be done on demand if the check starts failing in future.